app/models/site_setting.rb

```diff
@@ -1,7 +1,7 @@
 # frozen_string_literal: true
 
 class SiteSetting < ActiveRecord::Base
-  VALID_AREAS = %w[about embedding emojis flags fonts notifications permalinks]
+  VALID_AREAS = %w[about embedding emojis flags fonts notifications permalinks legal]
```

Comment:
I've been keeping this list alphabetised, but don't have particularly strong feelings about it. 🙂

```suggestion
  VALID_AREAS = %w[about embedding emojis flags fonts legal notifications permalinks]
```

Reply:
Yah fair enough, will do this!